About/former.md

```diff
@@ -0,0 +1,26 @@
+
+
+# Former Board Members:

```

Comment:
I think there is a separate Pull request to add the former board members. Please make sure to separate the different changes into different pull requests (i.e., different branches in your repo that are the source of the different pull requests).